apps/docs/app/src/routes/architecture/syner-integration/+page.md

```
@@ -0,0 +1,364 @@
+
+### 1.2 SSE Signing Bridge
+
+**`GET /api/syner/events`**
+
+Server-Sent Events stream for pushing signing requests to Syner.
+
+Authentication: Device JWT in `Authorization: Bearer <token>` header.
+
+Events:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)